fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3474,4 +3474,32 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = false)
     public static int max_historical_automated_cluster_snapshot_jobs = 100;
+
+    /**
+     * The URL to a JWKS service or a local file in the conf dir
```

Comment:
The comments can be added in to the comment field like
@ConfField(mutable = fase, comment = "xxx");
So that `admin show config` will see the usage of this config param.